project/src/services/api.ts

```diff
@@ -0,0 +1,13 @@
+import axios, {AxiosInstance} from 'axios';
+
+const BACKEND_URL = 'https://10.react.pages.academy/six-cities';
+const REQUEST_TIMEOUT = 5000;
+
+export const createAPI = (): AxiosInstance => {
```

Comment:
```suggestion
export const createAPI = ()=> {
```
а точно надо тип возврата писать?

Reply:
я старался следовать коду академии и в учебном проекте createApi типизировали